testnet/stacks-node/Cargo.toml

```diff
@@ -39,7 +38,7 @@ tikv-jemallocator = {workspace = true}
 [dev-dependencies]
 ring = "0.16.19"
 warp = "0.3.5"
-tokio = "1.15"
+tokio = { version = "1.15", features = ["rt-multi-thread"] }
```

Comment:
`Runtime::new()` only exists when this feature is enabled, the tests wouldn't compile for me otherwise, hence why I added the `rt-multi-thread` feature.